analyzers/src/SonarAnalyzer.CSharp/Rules/ForLoopCounterChanged.cs

```diff
@@ -43,19 +44,19 @@ public sealed class ForLoopCounterChanged : SonarDiagnosticAnalyzer
         private sealed class SideEffectExpression
         {
             public IImmutableList<SyntaxKind> Kinds { get; set; }
```

Comment:
```suggestion
            public ImmutableHashSet<SyntaxKind> Kinds { get; set; }
```

Reply:
@mary-georgiou-sonarsource What about these changes?

Reply:
All applied. 
I'm sorry for the confusion.